content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Configuring Gateway Network Topology (Experimental)
+description: How to configure gateway network topology (experimental).
+weight: 60
+keywords: [traffic-management,ingress,gateway]
+---
+
+## (Experimental) Configuring network topologies
+
+Istio provides the ability to manage settings like X-Forwarded-For (XFF) and X-Forwarded-Client-Cert (XFCC), which are
```

Comment:
I link to what an XFF an XFCC (IETF RFCs if any exist or any other documentation)would be immensely helpful.

Reply:
Added.